src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
+
+  const getPartner = () => {
+    if (isMarried) {
+      if (sex === 'm') {
+        return `${partnerName} is my wife`;
+      }
+
+      if (sex === 'f') {
+        return `${partnerName} is my husband`;
+      }
+    }
+
+    return 'I am not married';
+  };
```

Comment:
Actually this function looks too complicated, you can create a partner variable instead

const partner = sex === 'm'
    ? 'wife'
    : 'husband';

And then use this variable inside JSX

       {isMarried
          ? `${partnerName} is my ${partner}`
          : 'I am not married'
        }